api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1884,6 +1888,157 @@ public void removeObject(String bucketName, String objectName)
   }
 
 
+  private List<DeleteError> removeObject(String bucketName, List<DeleteObject> objectList)
+    throws InvalidBucketNameException, NoSuchAlgorithmException, InsufficientDataException, IOException,
+           InvalidKeyException, NoResponseException, XmlPullParserException, ErrorResponseException,
+           InternalException {
+    Map<String,String> queryParamMap = new HashMap<>();
+    queryParamMap.put("delete", "");
+
+    DeleteRequest request = new DeleteRequest(objectList);
+    HttpResponse response = executePost(bucketName, null, null, queryParamMap, request);
+
+    String bodyContent = "";
+    Scanner scanner = new Scanner(response.body().charStream());
```

Comment:
Done.

Many ways to do that http://stackoverflow.com/questions/309424/read-convert-an-inputstream-to-a-string